tools/packaging/static-build/coco-guest-components/build.sh

```diff
@@ -39,17 +39,17 @@ docker pull ${container_image} || \
 	 push_to_registry "${container_image}")
 
 # Temp settings until we have a matching TEE_PLATFORM
-TEE_PLATFORM=""
-RESOURCE_PROVIDER="kbs,sev"
+TEE_PLATFORM="${TEE_PLATFORM:-all}"
```

Comment:
If we default to `all` here, are we not going to end up trying to build the tpm attesters in `s390x` and fail here, or is there an override that sets `TEE_PLATFORM=se` if the arch it s390x, that I'm missing?

Reply:
You are right. That should make the s390x build failing. There is no override setting `TEE_PLATFORM=se` in the s390x pipeline. It must be `TEE_PLATFORM="${TEE_PLATFORM:-}"`.  Thanks for pointing it out.